README.md

```diff
@@ -652,6 +652,24 @@ const phoneNumber = DeviceInfo.getPhoneNumber();
 
 ---
 
+### getPowerState()
```

Comment:
Just added it. I didn't put anything under the "available since" column since I'm not sure what release this will actually land in.

On the Android note it does look like [they have a similar API](https://developer.android.com/training/monitoring-device-state/battery-monitoring). Unfortunately I don't have a physical Android device to test on, so while I could take a stab at it I'm not sure I'd be the best one to push it across the finish line :)

Reply:
this looks nice, and we can't all do everything, normally, happy to have an iOS dev, and I need the Java stuff I'll just have to code it up...